dataprofiler/tests/profilers/test_categorical_column_profile.py

```diff
@@ -155,39 +155,32 @@ def test_true_categorical_report(self):
         profile = CategoricalColumn(df_categorical.name)
         profile.update(df_categorical)
         report = profile.profile
-        six.assertCountEqual(
-            self, ['categorical', 'statistics', 'times'], report)
-        self.assertTrue(report["categorical"])
-        six.assertCountEqual(
-            self,
-            ['unique_count', 'unique_ratio', 'categories', 'categorical_count'], report['statistics']
-        )
-        self.assertEqual(3, report["statistics"]["unique_count"])
-        self.assertEqual(0.25, report["statistics"]["unique_ratio"])
-        self.assertCountEqual(
-            ["a", "b", "c"], report["statistics"]["categories"]
+        self.assertIsNotNone(report["times"])
+        report.pop("times")
```

Comment:
this could be a bit dangerous as while the test would fail, it would fail due to a code exception and not an assert.
This would happen in the case where times was not a key in the dict.

Hence, why I suggest reducing the two lines to: `self.assertIsNotNone(report.pop("times", None))`